### ISSUE TITLE
ClassFormatError: Unknown constant tag for large namespace

### ISSUE THREAD

**Issue by mfikes** [cursive-ide/cursive#680]
I am able to deterministically cause a failure to do "Tools > REPL > Load File in REPL" when the namespace is rather large. (This is with ClojureScript, "browser-REPLd" into embedded JavaScriptCore on iOS, doing this with `simple-brepl` and `Weasel`.)

```
Loading test/cljs/classroom_checkout/test1.cljs... done
CompilerException java.lang.ClassFormatError: Unknown constant tag 116 in class file     classroom_checkout/test1$eval12170, compiling:(/private/var/folders/m0/161fm8fx069fk_fny08nrmkm0000gp/T/form-init4953471926872611372.clj:1:1131) 
```

I did some investigation and found that it is essentially the same as the issue discussed [here](https://code.google.com/p/counterclockwise/issues/detail?id=429), which involves exceeding the 65535 byte limit for a compiled `.class` file.

I initially discovered this when trying to load the new (rather large) ClojureScript core test namespace into my REPL and subsequently discovered the thread mentioned above and confirmed that the problem occurs for the contents of the `foo.clj` test file attached to that thread, and it goes away when I delete a few lines from the namespace. (I say "contents" because I'm dumping the contents of that file, minus its `ns` form, into a `.cljs` file.)

In short, it appears that something in the toolchain is causing a large `.class` file to be generated.

I found that, if I issue a `(load-file "/path/to/myfile.cljs") form in the REPL it works, with the ClojureScript tests being loaded, and then runnable (several thousand of them run).

I also found that this, in general doesn't occur for loading a large conventional Clojure namespace with Cursive, so this makes me think that the problem could perhaps be instead `simple-brepl` or `Weasel`.

I'm certainly willing to do further investigation, as this may be hard to reproduce without the environment setup. But my first question would be: Does Cursive try to load a ClojureScript namespace as one large form, as was mentioned in the CounterClockwise thread?
